### ISSUE TITLE
Please read & provide the following

### ISSUE THREAD

**Issue by zxyyxzz** [facebookresearch/detectron2#2835]
> misinterpret

**Thanks for your reply
sorry, i just post a wrong code yesterday:

the below code is correct, the x and y don't misinterpret now, but the roi_align still outputs zeros, do you know what happened, can u help me?

hope u can reply me~~~~
thanks!!~~~**

`    output_size = (7,7)
    spatial_scale = 1/4
    sampling_ratio = 2

    x = torch.zeros((1,1,117,117), dtype=torch.float)
    x[:,:,53:88,89:102]=100.0
    axis = torch.nonzero(x)
    a, _ = axis.max(dim=0)
    b, _ = axis.min(dim=0)

    rois = torch.tensor([
        [0.0, 53.0, 89.0, 88.0, 102.0],
    ])

    ya = roi_align(x, rois, output_size)
    print(ya)`

tensor([[[[0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.],
          [0., 0., 0., 0., 0., 0., 0.]]]])

Process finished with exit code 0

**Comment by zxyyxzz** [comment]
@ppwwyyxx

**Comment by ppwwyyxx** [comment]
The x and y are still wrong in the code.

Since the code only uses torchvision, it is not related to detectron2. Please ask in places like pytorch forum.
